app/components/WardTile.tsx

```diff
@@ -0,0 +1,71 @@
+import { tw } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import type { FunctionComponent } from 'react'
+import Bed from '../icons/Bed'
+import { PillLabelBox } from './PillLabelBox'
+
+type WardTileTranslation = {
+  edit: string
+}
+
+const defaultWardTileTranslations: Record<Languages, WardTileTranslation> = {
+  en: {
+    edit: 'Edit'
+  },
+  de: {
+    edit: 'Bearbeiten'
+  }
+}
+
+type WardDTO = {
+  name: string,
+  roomNames: string[],
+  bedCount: number,
+  unscheduledTasks: number,
+  inProgressTasks: number,
+  doneTasks: number
+}
+
+export type WardTileProps = {
+  ward: WardDTO,
+  isSelected: boolean,
+  tileClickedCallback?: () => void,
+  editClickedCallback?: () => void
+}
+
+const WardTile: FunctionComponent<PropsWithLanguage<WardTileTranslation, WardTileProps>> = ({
```

Comment:
remove `FunctionComponent<PropsWithLanguage<WardTileTranslation, WardTileProps>>`, you already have a type annotation for the props object, no need to specify this twice (the props object is the preferred way as react has some odd types sometimes, thus it is best to rely on inference to do the heavy lifting)